### DISCUSSION TITLE
Tools SQL

### DISCUSSION THREAD

**Discussion by mdcj0208** [sqlitebrowser/sqlitebrowser#3232] [General]
CREATE TABLE sales (
    purchase_num INT PRIMARY KEY,
    customer_id VARCHAR(50),
    item_code VARCHAR(50),
    date_of_purchase DATE
);
							
INSERT INTO sales (purchase_num, customer_id, item_code, date_of_purchase)
	VALUES
	(1, 'CUST-0001', 'A_1', '2016-03-01'),
	(2, 'CUST-0002', 'B_1', '2016-03-01'),
	(3, 'CUST-0003', 'A_1', '2016-04-01'),
	(4, 'CUST-0004', 'C_2', '2016-04-01');
	
/*DATE FORMAT = YYYY-DD-MM*/

CREATE TABLE customers (
    customer_id VARCHAR(100) PRIMARY KEY,
    first_name VARCHAR(50),
    last_name VARCHAR(50),
    email_address VARCHAR(100),
    complaints INT NULL
);
    
    INSERT INTO customers (customer_id, first_name, last_name, email_address, complaints)
	VALUES 
	('CUST-0001', 'Adam', 'Jones', 
	'adamj22@email.com', 2),
	('CUST-0002', 'Brandon', 'Harris',  
	'bharris123@email.com', 1),
	('CUST-0003', 'Nicole', 'Gray', 
	'nikkigrey99@email.com', 1),
	('CUST-0004', 'Ashley', 'Darver', 'ashdarv123@email.com', 0);
	
	CREATE TABLE items(
item_code VARCHAR(50) PRIMARY KEY,
item VARCHAR(50),
price DECIMAL(5,2)
);

INSERT INTO items( item_code, item, price)
VALUES
( 'A_1', 'Hammer', 46.89),
('A_2', 'Screwdriver', 23.79),
('B_1', 'Pliers', 14.99),
('B_2', 'Wrench', 31.49),
('C_1', 'Power Drill', 109.99),
('C_2', 'Screws', 8.99);

SELECT sales.customer_id, sales.item_code, customers.first_name, customers.last_name
FROM sales 
JOIN customers
ON sales.customer_id = customers.customer_id
JOIN items
ON items.item_code = sales.item_code;

**Comment by chrisjlocke** [comment]
I assume you're aware this isn't valid SQLite SQL.  I assume MySQL?

**Reply by mdcj0208** [reply]
Thanks for catching that. I didn't even realize.

**Reply by chrisjlocke** [reply]
Is there a point to it?
A discussion doesn't seem ... er .... discussing.

**Reply by scottfurry** [reply]
This looks like introduction to SQL course material. Helping with homework using a different RDBMS, to me, is out of scope to project.
